src/components/FormGroup.jsx

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 
 const FormGroup = (props) => {
-  const { label, ...rest } = props
+  console.log(props);
+  const { label, ...rest } = props;
   const { styles } = FormGroup;
 
+  function inputField() {
+    if (props.placeholder === 'what are you looking for?') {
```

Comment:
I think we are getting away from the issue here which is that we should have defined styles for the site rather than adding conditional styling props to components.